src/analog/message.ts

```diff
@@ -144,6 +145,30 @@ export const fromBytes = ( data: Uint8Array, config?: IMessageConfig ) => {
     return result;
 };
 
+export const fromFrame = ( frame: Array<number> | Uint8Array, config?: IMessageConfig ) => {
+    const frameContent = Frame.fromFrame(frame);
+
+    if ( !frameContent ) {
+        return [];
+    }
+
+    const messageData = fromBytes(new Uint8Array(frameContent), config);
+
+    return messageData.commands;
+};
+
+export const fromFrames = ( frames: Array<Array<number> | Uint8Array>, config?: IMessageConfig ) => {
+    const commands = [];
+
+    for ( const frame of frames ) {
+        const commandsInFrame = fromFrame(frame, config);
+
+        commands.push(...commandsInFrame);
+    }
+
+    return commands.flatMap( command => command.command);
```

Comment:
```suggestion
    return commands.flatMap(command => command.command);
```

Reply:
fixed

Reply:
Changes in analog/message.ts are removed from the request for now.